assets/openshift-state-metrics/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1beta2
```

Comment:
we've upgraded everything to use the apps/v1 API, can we update here as well? :)